extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/icon.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react'),
+    Utils = require('./utils');
+
+
+var Icon = React.createClass({
+  shouldComponentUpdate: function(nextProps, nextState) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "nextState". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)